canonicalwebteam/blog/djangoblog/tests.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
I don't think you need to have the tests file if no tests (even if it's just a bootstrap)

And the pattern we have for tests is to have a `tests` at root of the project. We could change this (more js style) but for now I would prefer to keep our patterns.